lib/api/services/ProductService.ts

```diff
@@ -1078,6 +1078,9 @@ export class ProductService extends Service {
   removeImage(id, options: { [key: string]: any } = {}) {
     const Image = this.app.models['ProductImage']
     const Product = this.app.models['Product']
+    const Variant = this.app.models['Variant']
```

Comment:
refactor `const Variant = this.app.models['Variant']`
to `const Variant = this.app.models['ProductVariant']`